Actors/ActorCounter/Scripts/install-sfctl.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -x
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+appPkg="$DIR/../CounterActorApplication"
+
+sfctl application upload --path $appPkg --show-progress
```

Comment:
I am wondering - where does this 'sfctl' come from? I had a look at the following documentations - 
https://docs.microsoft.com/en-us/azure/service-fabric/service-fabric-azure-cli-2-0 

Also, I installed this cli 2.0 and tested other couple of basic commands following the same approach, mentioned there. But I could not find 'sfctl'. It was 'az sf' everywhere. Can you just check once and let me know if I am missing anything,